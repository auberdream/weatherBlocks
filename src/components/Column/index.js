import styled from 'styled-components';

export default styled.div`
    display: grid;
    grid-template-rows: ${({ rows = 24 }) => `repeat(${rows}, 1fr)`};
    height: 100vh;
`;
