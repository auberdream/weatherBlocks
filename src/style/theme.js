export default {
  palette: {
    weather: {
      sunny: '#f72585',
      cloudy: '#b5179e',
      mist: '#7209b7',
      rain: '#560bad',
      thunder: '#3a0ca3',
      snow: '#3f37c9',
      sleet: '#4361ee',
      fog: '#4cc9f0',
    },
  },
  select: {
    background: '#FFF',
  },
};
