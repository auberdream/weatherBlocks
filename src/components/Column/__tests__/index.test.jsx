import React from 'react';
import renderer from 'react-test-renderer';
import { Grommet } from 'grommet';

import theme from '../../../style/theme';
import Column from '..';

it('Renders correctly', () => {
  const tree = renderer.create(
    <Grommet theme={theme}>
      <Column />
    </Grommet>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Renders correctly for different rows', () => {
  const tree = renderer.create(
    <Grommet theme={theme}>
      <Column rows={12} />
    </Grommet>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
