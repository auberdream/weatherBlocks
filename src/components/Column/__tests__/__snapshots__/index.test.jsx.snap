// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 djOnwZ"
>
  <div
    className="sc-bdfBwQ blkaHo"
  />
</div>
`;

exports[`Renders correctly for different rows 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 djOnwZ"
>
  <div
    className="sc-bdfBwQ imIjpl"
    rows={12}
  />
</div>
`;
